Bat.bat

```diff
@@ -3,6 +3,7 @@ Setlocal EnableDelayedExpansion
 
 Set "_path=%LocalAppData%\BatCenter"
 Set "Original_Path=%path%"
+Set "CallDir=%CD%"

```

Comment:
You do not need this, if the path is included in the "PATH" Variable itself. @GroophyLifefor